### PR TITLE
Feature/coverage

### PR DESCRIPTION
**Pull request by trasher**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes

### CODE REVIEW

File: .github/workflows/coverage.yml [hunk 1]
```diff
@@ -0,0 +1,99 @@
+name: "GLPI test code coverage"
+
+on:
```
Comment (trasher) [inline .github/workflows/coverage.yml:3]: must be adapted to be run by cron, once per week I guess

**Comment by trasher** [comment]
My first idea was to generate all HTML reports from atoum and made them available from a gh action artifact; but running tests on all directories at once fails here (and it works locally).

So I've ended to generate several clover reports in different directories (almost the same was done when we tried to use scritinizer coverage).
I did not really found an easy way to get something readable from xml reports; codecov proposes that from an action; that what I've used.

**Comment by trasher** [comment]
Example result at https://codecov.io/gh/glpi-project/glpi/tree/51e3434d29248606039a70330cf8b7847ae3b119/inc

**Comment by cedric-anne** [comment]
I've rebased on 9.5 branch (to be able to use the cron trigger, that is only active on default branch), and review the workflow configuration to use existing scripts.

**Review by cedric-anne — APPROVED** [review]

**Pull request merged**